src/measurement/sev_hashes.rs

```diff
@@ -18,6 +20,14 @@ use crate::error::*;
 
 type Sha256Hash = [u8; 32];
 
+#[cfg(all(not(feature = "openssl"), feature = "crypto_nossl"))]
```

Comment:
Same thing 
```suggestion
#[cfg(feature = "crypto_nossl")]
```

Reply:
Again, my mistake. I asked @jakubDoka forgetting that it won't compile with both features.